build.sbt

```diff
@@ -118,6 +119,12 @@ lazy val dev = project
     libraryDependencies ++= Seq(
       "com.github.scopt" %% "scopt" % "4.0.1"
     ),
-    publish / skip := true
+    publish / skip := true,
+      assembly / assemblyMergeStrategy := {
```

Comment:
The indentation looks a little off here. But also, it might be worth expanding on why this is being duplicated here. If it is really required, is it worth finding a way to re-use this across `dev` and `anghammarad` to avoid the duplication?

Reply:
We tried but did not quickly succeed, will ask you about it Wednesday.